prism_compile.c

```diff
@@ -1206,6 +1222,27 @@ pm_compile_defined_expr(rb_iseq_t *iseq, const pm_defined_node_t *defined_node,
 #undef PUSH_VAL
 }
 
+void
+pm_compile_defined_expr(rb_iseq_t *iseq, const pm_node_t *node, LINK_ANCHOR *const ret, const uint8_t *src, bool popped, pm_scope_node_t *scope_node,  NODE dummy_line_node, int lineno, bool in_condition)
+{
+    LABEL *lfinish[3];
+    LINK_ELEMENT *last = ret->last;
+
+    lfinish[0] = NEW_LABEL(lineno);
+    lfinish[1] = 0;
+    lfinish[2] = 0;
```

Comment:
I think we don't need `lfinish[2]` and can instead make this an array with 2 elements

Reply:
updated thanks!